test/e2e/storage/utils/utils.go

```diff
@@ -104,6 +96,9 @@ func TestKubeletRestartsAndRestoresMount(ctx context.Context, c clientset.Interf
 	ginkgo.By("Restarting kubelet")
 	KubeletCommand(ctx, KRestart, c, clientPod)
 
+	ginkgo.By("Wait 20s for the volume to become stable")
+	time.Sleep(20 * time.Second)
```

Comment:
Hmm. Magic timeouts have been great (not!) source of test flakes. Can you elaborate on why 20 seconds where chosen? How certain is it that this value works reliably? How does it affect the runtime of tests?

Reply:
This sleep is moved from `KubeletCommand()` just above it. We are asserting that something is not happening, so we need a timeout.

I choose 20 seconds because the original code in KubeletCommand():
```go
framework.Logf("Noticed that kubelet PID is changed. Waiting for 30 Seconds for Kubelet to come back")
time.Sleep(30 * time.Second)
```
The sleep now starts after node become ready, so (20s + WaitForNodeToBeReady) should roughly equal to the original 30s.